core/src/components/textarea/textarea.scss

```diff
@@ -179,6 +147,17 @@
   }
 }
 
+.native-textarea,
+:host(.legacy-textarea) .textarea-legacy-wrapper::after {
+  @include padding(var(--padding-top), var(--padding-end), var(--padding-bottom), var(--padding-start));
```

Comment:
This is already set here https://github.com/ionic-team/ionic-framework/blob/FW-2764/core/src/components/textarea/textarea.scss#L284

Also why do we set `text-inherit()`?